test/end-to-end/paths/probate/gopSingleExecutor.en.test.js

```diff
@@ -1,154 +1,161 @@
 'use strict';
 
-const taskListContent = require('app/resources/en/translation/tasklist');
+const taskListContentEn = require('app/resources/en/translation/tasklist');
+const taskListContentCy = require('app/resources/cy/translation/tasklist');
 const TestConfigurator = new (require('test/end-to-end/helpers/TestConfigurator'))();
 const optionYes = '';
 const ihtPost = '';
 const optionNo = '-2';
 const bilingualGOP = false;
 const uploadingDocuments = false;
 const config = require('config');
+const languages = ['en', 'cy'];
 
-Feature('GOP-Single Executor flow...').retry(TestConfigurator.getRetryFeatures());
+Feature('GOP-Single Executor').retry(2);
 
```

Comment:
Be good to retain getting this from config - when I run locally I always set thes in config to 0 then undo my config change as I don't want the retries for local dev